### ISSUE TITLE
CC005 - new false positives after v2.68.0

### ISSUE THREAD

**Issue by dangrasso** [apigee/apigeelint#551]
```
error CC005 : Invalid Condition - cannot parse 
```
   
example
```
<Condition>notValidJson == true</Condition>
```

This is likely a regression due to the fix in https://github.com/apigee/apigeelint/pull/544/files#diff-da821e73288e51a008df7f71c9da3c764aa1cf983978ef7ecdf72881c9a72aaeR483

Would an option be to explicitly replace only `not(` and `not ` but not everything starting with `not`?

**Comment by DinoChiesa** [comment]
Can you check to see if the false positives continue, with the forthcoming release?

@ssvaidyanathan will deliver a release shortly I think.   Or you can  pull latest main, and try that.

**Comment by ssvaidyanathan** [comment]
v2.69.0 is released

**Comment by DinoChiesa** [comment]
@dangrasso did you confirm that v2.69.0 works for you?  I would expect that it would, but I'd love to hear your confirmation.

**Comment by dangrasso** [comment]
Thanks for checking @DinoChiesa. It actually doesn't work. I don't understand why this time. 

The failure points to the same condition:
```
<Condition>notValidJson == true</Condition>
```

I tried your new regex directly on my condition and as expected it doesn't replace the `not` this time. But still CC005 is marked as failed for that condition block. I also tried switching between different versions:

2.67.0 Pass 🟢 
2.68.0 Fail 🔴 
2.69.0 Fail 🔴

**Comment by dangrasso** [comment]
Actually I know why it still fails @DinoChiesa @ssvaidyanathan I see that this replacement still happens a few lines above: https://github.com/DinoChiesa/apigeelint/blob/bc113b92f6f39ae7868f27f8b6f4480c32898514/lib/package/TruthTable.js#L484

The * quantifier will match on 0 or more occurrences of the space, so any word starting with not will be affected.

You probably want to match only on `^NOT\s+`

**Comment by DinoChiesa** [comment]
Yes, thanks for the diagnosis.

**Comment by DinoChiesa** [comment]
How 'bout now , Dan?  Works?
